common/djangoapps/student/helpers.py

```diff
@@ -230,3 +235,38 @@ def get_next_url_for_login_page(request):
         # be saved in the session as part of the pipeline state. That URL will take priority
         # over this one.
     return redirect_to
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
I would move it back where it came from. I definitely think it belongs in the user_api package.